add-on/chrome/bootstrap.js

```diff
@@ -1404,10 +1404,8 @@ function startup(data) {
     return;
   }
 
-  // register about: handlers
-  AboutLoop.conversation.register(); // eslint-disable-line no-undef
-  AboutLoop.panel.register(); // eslint-disable-line no-undef
-  AboutLoop.toc.register(); // eslint-disable-line no-undef
+  Services.ppmm.loadProcessScript(
+    "chrome://loop/content/modules/loop-content-process.js", true);
```

Comment:
According to [devmo](https://developer.mozilla.org/en-US/Firefox/Multiprocess_Firefox/Message_Manager/Process_scripts) process scripts work the same way as frame scripts. Can you therefore set this up with a const url in a similar way to FRAME_SCRIPT please?

That will help us when we get to proper restartless.